<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>編輯既有工程</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="qtSearchNumberRadioButton">
         <property name="text">
          <string>搜尋工程編號</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="qtSearchNameRadioButton">
         <property name="text">
          <string>搜尋工程名稱</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="qtSearchLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="qtTableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="qtExitPushButton">
     <property name="text">
      <string>離開</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
